### DISCUSSION TITLE
Will Native Forms be supported in Maui?

### DISCUSSION THREAD

**Discussion by jdunaway** [dotnet/maui#3815] [Q&A]
We have a Xamarin native iOS/Android project that we are considering gradually migrating to Xamarin Forms or Maui.  Will the concept of [Native Forms](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/platform/native-forms) be supported in Maui?  It looks like the CreateViewController method for iOS exists in [src/Compatibility/Core/src/iOS/PageExtensions.cs](https://github.com/dotnet/maui/blob/2e70dc6e072090fb42e9f03380a30040da8ce8b6/src/Compatibility/Core/src/iOS/PageExtensions.cs) in the Maui project and CreateSupportFragment for Android exists in [src/Compatibility/Core/src/Android/AppCompat/PageExtensions.cs](https://github.com/dotnet/maui/blob/c6efff91de8fe30fadbcdfff24f7b9af96dc0c91/src/Compatibility/Core/src/Android/AppCompat/PageExtensions.cs).  So does that mean Native Forms is still supported in Maui even though it exists in the Compatibility folder?  Or is there another concept in Maui that is replacing Native Forms?  We want to make sure that if we start working with Native Forms now that we won't have any issues when we need to migrate the project to Maui in the future.

**Comment by saamerm** [comment]
@jdunaway It seems like Native Forms is supported, have you tested it out at all? I have added an enhancement request as well https://github.com/dotnet/maui/issues/9606
